Source/Core/DolphinQt2/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(SRCS
 	Host.cpp
 	RenderWidget.cpp
 	Resources.cpp
+	Translate.cpp
```

Comment:
Not a big deal, but to be consistent this should be Translator.cpp/Translator.h.